SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -238,18 +243,32 @@ void USpatialReceiver::OnRemoveEntity(const Worker_RemoveEntityOp& Op)
 		return;
 	}
 
-	RemoveActor(Op.entity_id);
+	if (LoadBalanceEnforcer != nullptr)
```

Comment:
Do this before the async load check.

Reply:
Otherwise LoadBalanceEnforcer will never clean up the entity.

Reply:
now addressed